person.rb

```diff
@@ -1,4 +1,5 @@
 require './nameable'
+require './rental'
```

Comment:
- [ ] Please add `@rentals` to the `attr_accessor` just like the way you did for the `name` and `age` in line 6 of this file.